### DISCUSSION TITLE
Get mojo in testnet?

### DISCUSSION THREAD

**Discussion by neal-zhu** [Chia-Network/chia-blockchain#6536] [Support]
https://faucet.chia.net/ returns appears invalid address with testnet address? So how can I get mojo in testnet to test pool?

**Comment by loppefaaret** [comment]
For testnet7 http://chia-faucet.com/

**Comment by BrandtH22** [comment]
@neal-zhu , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
